src/sentry/models/dashboard_permissions.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import annotations
+
+from django.db import models
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import Model, region_silo_model
+from sentry.db.models.base import sane_repr
+from sentry.db.models.fields.hybrid_cloud_foreign_key import HybridCloudForeignKey
+
+
+@region_silo_model
+class DashboardPermissions(Model):
+    """
+    Edit permissions for a Dashboard.
+    """
+
+    __relocation_scope__ = RelocationScope.Organization
+
+    created_by_id = HybridCloudForeignKey("sentry.User", on_delete="CASCADE")
+    is_creator_only_editable = models.BooleanField(default=False)
```

Comment:
Will there be more fields than this? Having an additional table for a 1:1 record with a small number of fields seems more complex than adding the permission relevant fields to dashboards table.

Reply:
Yup, the plan is to add more fields. This will include team-based edit permissions and sharing options for creating shareable dashboard links.

Reply:
In the long term would all dashboards have a `dashboard_permission` record? If so we could fold all of the permission columns onto `dashboard` and have a simpler storage model. If only some dashboards will have permission records than a separate table makes sense, and is preferable to a chunk of nullable columns.

Reply:
The long term plan is to not enable permissions by default for a dashboard. Essentially, we want this to remain an optional feature to be used by those who really want to restrict their dashboards being edited/shared.